### ISSUE TITLE
Is it possible to support CMake build ?

### ISSUE THREAD

**Issue by peerless2012** [fribidi/fribidi#215]

**Comment by TheNicker** [comment]
@peerless2012 , try #225 let me know if it works.

**Comment by TheNicker** [comment]
Yes, it is possible, but probably won't be supported in this repo, please refer to the following:
https://github.com/fribidi/fribidi/pull/225#issuecomment-3092539331

Ticket can be closed.

@peerless2012, CMake implementation works for me for many years, #225 is a rebased implementation of it. 
I'm not sure it is completed, or solves all cases, plus in this CMake build there's a sub CMake build which is not implemented in an ideal way. if you would like to continue and improve the work on CMake please refer to https://github.com/fribidi-cmake/fridibi

**Comment by peerless2012** [comment]
OK, thanks for your work.
